testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -2883,19 +2900,26 @@ public void testTimestampHandling() throws IOException {
         HttpStatus.SC_CREATED);
 
     // insert a row, ensuring we can use literals for numeric values
-    String timestamp2 = String.valueOf(Instant.now().toEpochMilli());
+    String timestamp2 = Instant.now().toString();
     Map<String, Object> row2 = new HashMap<>();
     row2.put("hour_created", timestamp2);
     row2.put("serial_number", "ZXY765");
     row2.put("machine_code", "MNO432");
     row2.put("part_name", "Adapter");
     row2.put("part_number", "QRS246");
-    row2.put("last_inspected_at", timestamp2);
+    row2.put("created_at", "23:19");
+    row2.put("last_inspected_at", "2020-02-01");
     row2.put("times_inspected", 5);
     row2.put("est_unit_cost", 38.95);
     row2.put("est_unit_cost_updated", timestamp2);
     row2.put("inspection_notes", "frayed cable");
     row2.put("mean_failure_time_hours", 5917321.12334);
+    row2.put("rating", (float) 92.6);
+    row2.put("stars", (byte) 4);
+    row2.put("likes", (short) 926);
+    row2.put("test_runs", BigInteger.ONE);
+    row2.put("source_ip", "FE80:CD00:0:CDE:1257:0:211E:729C");
+    row2.put("description", (new String("AQID//7gEiMB")).getBytes());
```

Comment:
No need to use String constructor. Also: should pass encoding (although being ASCII not a big deal), `StandardCharsets.UTF_8` or such.